packages/framework/ai-collab/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+	"extends": "../../../common/build/build-common/tsconfig.node16.json",
+	"include": ["src/**/*"],
+	"exclude": ["src/test/**/*"],
+	"compilerOptions": {
+		"rootDir": "./src",
+		"outDir": "./lib",
+		"exactOptionalPropertyTypes": false,
+		"noUnusedLocals": false,
```

Comment:
Production code has unused locals? If there are, it is probably better to suppress at particular sites.

Reply:
It was here probably just due to copy paste, build runs fine without it.